Library/Homebrew/test/test_patching.rb

```diff
@@ -122,7 +122,9 @@ def test_single_patch_dsl_with_incorrect_strip
             url PATCH_URL_A
             sha256 PATCH_A_SHA256
           end
-        end.brew(&:patch)
+        end.brew do |f, _staging|
+          f.patch
+        end
```

Comment:
For both brevity and clarity, I would make that `end.brew { |formula, _staging| formula.patch }`. (The shorter `f` seems also acceptable, given how widely that is used in our code base.)